ecosystem/sep-0006.md

```diff
@@ -688,7 +906,7 @@ Name | Type | Description
 `account` | string | The stellar account ID involved in the transactions.
 `no_older_than` | UTC ISO 8601 string | (optional) The response should contain transactions starting on or after this date & time.
 `limit` | int | (optional) The response should contain at most `limit` transactions.
-`kind` | string | (optional) The kind of transaction that is desired. Should be either `deposit` or `withdrawal`.
+`kind` | string | (optional) The kind of transaction that is desired. Should be either `deposit`, `deposit-exchange`, `withdrawal` or `withdrawal-exchange`.
```

Comment:
This has me wondering -- what if a client wants both types of deposits? Are we just going to require they make 2 API calls?

Reply:
Good point, that wasn't a problem when there were only two kinds...

If we want to allow subsets we could either add a new field called `kinds` or update the `kind` description to accept a comma-separated list, like "deposit,deposit-exchange" for instance. I'm leaning towards the comma-separated option, WDYT?

Reply:
You could allow multiple `kind` params, i.e.g `/transactions?kind=deposit&kind=deposit-exchange`, which is more likely to be backwards compatible with existing services. This would be my preference for this reason, and because you're leveraging URL queries natural expression of arrays without introducing comma-separated lists or new params.

Reply:
Applied Leigh's suggestion in https://github.com/stellar/stellar-protocol/pull/973/commits/31e833b0bd2a2726086c1a03ef2ac14ee9b7dd9f.